Adding log section (#4824)
DataDog/integrations-core: squid/README.md

### Configuration

> 1. Edit the `squid.d/conf.yaml`, in the `conf.d/` folder at the root of your [Agent's configuration directory][3] to start collecting your Squid metrics and [logs](#log-collection). See the [sample squid.d/conf.yaml][4] for all available configuration options.

Reviewer: ```suggestion
1. Edit the `squid.d/conf.yaml`, in the `conf.d/` folder at the root of your [Agent's configuration directory][3] to collect your Squid metrics and [logs](#log-collection). See the [sample squid.d/conf.yaml][4] for all available configuration options.
```

2. [Restart the Agent][5].

#### Log collection

**Available for Agent >6.0**

> 1. Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:

Reviewer: ```suggestion
1. Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file:
```

    ```yaml
      logs_enabled: true
    ```

> 2. Add this configuration block to your `squid.d/conf.yaml` file to start collecting your Squid Logs:

Reviewer: ```suggestion
2. Uncomment and edit this configuration block at the bottom of your `squid.d/conf.yaml` file:
```
